capstone/src/main/java/com/google/univiz/api/GeneralCollegeData.java

```diff
@@ -0,0 +1,21 @@
+package com.google.univiz.api;
```

Comment:
This interface now belongs to a different package like `package com.google.univiz.scorecard;`